modules/datastore/tests/data/query/groupByQuery.json

```diff
@@ -0,0 +1,40 @@
+{
+    "resources": [
+        {
+            "id": "asdf",
+            "alias": "t"
+        }
+    ],
+    "properties": [
+        {
+            "resource": "t",
+            "property": "prop"
+        },
+        {
+            "alias": "sum",
+            "expression": {
+                "operator": "sum",
+                "operands": [
+                    {
+                        "resource": "t",
+                        "property": "summable"
+                    }
+                ]
+            }
+        }
+    ],
+    "conditions": [
+        {
+            "resource": "t",
+            "property": "filterable",
+            "value": "value",
+            "operator": "="
+        }
+    ],
+    "groups": [
```

Comment:
@clayliddell the more I think about it the more I think, semantically, "groupings" is the right word to use here, not "groups." Would you be open to changing this?

Reply:
Yeah, I like that better!